certified-connectors/Dime.Scheduler/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4404 @@
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/assignment"
+            },
+            "description": "JSON request body containing the contents of the requested entity to be imported."
+          }
+        ],
+        "responses": {
+          "200": {
+            "$ref": "#/responses/200"
+          },
+          "400": {
+            "$ref": "#/responses/400"
+          },
+          "500": {
+            "$ref": "#/responses/500"
+          },
+          "default": {
+            "$ref": "#/responses/default"
+          }
+        }
+      }
+    },
+    "/caption": {
+      "post": {
+        "summary": "Add, update or remove a caption",
+        "description": "Add, update or remove a caption.",
+        "operationId": "Caption",
+        "parameters": [
+          {
+            "name": "ds-append",
+            "in": "header",
+            "description": "True to create or update the record, false to remove.",
+            "type": "boolean",
+            "x-ms-summary": "Append"
+          },
+          {
+            "name": "caption",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/caption"
+            },
+            "description": "JSON request body containing the contents of the requested entity to be imported."
+          }
+        ],
+        "responses": {
+          "200": {
+            "$ref": "#/responses/200"
+          },
+          "400": {
+            "$ref": "#/responses/400"
+          },
+          "500": {
+            "$ref": "#/responses/500"
+          },
+          "default": {
+            "$ref": "#/responses/default"
+          }
+        }
+      }
+    },
+    "/container": {
+      "post": {
+        "summary": "Add, update or remove a container",
+        "description": "Add, update or remove a container.",
+        "operationId": "Container",
+        "parameters": [
+          {
+            "name": "ds-append",
+            "in": "header",
+            "description": "True to create or update the record, false to remove.",
+            "type": "boolean",
+            "x-ms-summary": "Append"
+          },
+          {
+            "name": "container",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/container"
+            },
+            "description": "JSON request body containing the contents of the requested entity to be imported."
+          }
+        ],
+        "responses": {
+          "200": {
+            "$ref": "#/responses/200"
+          },
+          "400": {
+            "$ref": "#/responses/400"
+          },
+          "500": {
+            "$ref": "#/responses/500"
+          },
+          "default": {
+            "$ref": "#/responses/default"
+          }
+        }
+      }
+    },
+    "/filterGroup": {
+      "post": {
+        "summary": "Add, update or remove a filter group",
+        "description": "Add, update or remove a flter group.",
```

Comment:
@hbulens Please make sure that descriptions are free from grammatical errors or typos. please change "flter" to "filter".
Thank you so much.

Reply:
Well spotted, thanks for pointing that out. See commit [9ee7496](https://github.com/microsoft/PowerPlatformConnectors/pull/2091/commits/9ee7496977c9b66097f89e63a13bcf590814a7cb) for the change.